app/src/main/kotlin/com/twain/interprep/presentation/ui/modules/notes/ViewNotesScreen.kt

```diff
@@ -89,6 +94,17 @@ fun ViewNotesScreen(
                             )
                         }
                     }
+                    item {
+                        if (viewModel.notes.isEmpty()) {
+                            Spacer(modifier = Modifier.height(dimensionResource(id = R.dimen.dimension_75dp)))
```

Comment:
Is there any other way to make FullScreenEmptyState vertically center aligned instead of hardcoded space of 75 dp?